### ISSUE TITLE
Hotkey conflict

### ISSUE THREAD

**Issue by KewaiiGamer** [mezz/JustEnoughItems#215]
I am used to press "R" to sprint and with NEI I could sprint and show recipes with R however with JEI this cannot be done. So maybe incorporate the controls the same way NEI did because it is really annoying having to either switch the "Show recipe" key or the "sprint" to somewhere else when I am already to used to have both as "R"

On 1.8.9 I switched the sprint key to "F" but on 1.9 the "F" is binded to "Swap items in Hands" (Dual wielding) and it gets really annoying

**Comment by Prospector** [comment]
Yes it should be configurable, but how is this an issue? When you press R and your inventory is closed you will sprint, and when it is opened you can't sprint anyway so it'll use the recipe right?

**Comment by mezz** [comment]
Prospector, it uses Forge's hotkey registration so it counts as a "conflict" unfortunately. 
I'm waiting on this, which should fix everything: https://github.com/MinecraftForge/MinecraftForge/pull/2234

**Comment by KewaiiGamer** [comment]
Also checked that FORGE link and it looks like I will have to wait. It's not just JEI. it's the sprint key itself that can't be used for anything else. The sneak key also has this problem

**Comment by mezz** [comment]
I wrote a new PR for Forge that will solve this issue, https://github.com/MinecraftForge/MinecraftForge/pull/2674

**Comment by Prospector** [comment]
I thought you were on the forge team?

**Comment by mezz** [comment]
I don't think that means direct push access for new features... still want to get feedback and make sure it's a good idea.

**Comment by ghost** [comment]
Here is the log file for the error that I keep getting. This all happens on startup.

http://paste.ee/p/UPprQ

**Comment by mezz** [comment]
You need the very latest Forge, I guess I didn't update the required version in JEI.

**Comment by ghost** [comment]
I seem to be getting the same error on the most recent version of forge, too. Forge 1827

http://paste.ee/p/xQr8b

**Comment by TehNut** [comment]
The most recent Forge is 1834.

**Comment by mezz** [comment]
Updated latest JEI to say it requires Forge 1832+.

**Comment by ghost** [comment]
Thanks! I had to refresh MultiMC to see the newest version. It works now.
